### PR TITLE
feat(t05): UI for job detail

### PR DESCRIPTION
**Pull request by vutiendanhvgu01**
t05fix7.surge.sh

### CODE REVIEW

**Comment by Tiencmfaba** [comment]
I have seen you are using justify-content: center; and display: flex,
ca we use common for this ?

File: src/redux/reducer/JobReducer.tsx [hunk 1]
```diff
@@ -40,5 +46,14 @@ export const jobAPI = () => {
     dispatch(getJobAction(result.data));
   };
 };
+export const getJobDetailAPI = (id: number) => {
+  return async (dispatch: DispatchType) => {
+    const result = await axios.get(
+      `${process.env.REACT_APP_API_BASE_URL}/job/${id}`
+    );
 
+    dispatch(getJobDetailAction(result.data));
```
Comment (Tiencmfaba) [inline src/redux/reducer/JobReducer.tsx:55]: should to wrap if(result.data), right ?

File: yarn.lock [hunk 2]
```diff
@@ -9877,7 +9884,7 @@ prompts@^2.0.1:
     kleur "^3.0.3"
     sisteransi "^1.0.5"
 
-prop-types@^15.6.2:
+prop-types@^15.6.2, prop-types@^15.8.1:
```
Comment (Tiencmfaba) [inline yarn.lock:9887]: why do you upgrade version ?

**Comment by Tiencmfaba** [comment]
please, remove the lib don't need

File: src/containers/Detail/Detail.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+const Detail = (props: Props) => {
+  const { jobDetail } = useSelector((state: RootState) => {
+    return state.JobReducer;
+  });
+  const dispatch: DispatchType = useDispatch();
+  const params = useParams();
+  useEffect(() => {
+    if (params.id !== undefined) {
+      console.log("paramsid", params.id);
```
[the remark this answers is not in the document]
Comment (vutiendanhvgu01) [reply src/containers/Detail/Detail.tsx]: I have fixed

File: src/containers/Detail/Detail.tsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+type Props = {};
+
+const Detail = (props: Props) => {
+  const { jobDetail } = useSelector((state: RootState) => {
+    return state.JobReducer;
+  });
+  const dispatch: DispatchType = useDispatch();
+  const params = useParams();
+  useEffect(() => {
+    if (params.id !== undefined) {
```
[the remark this answers is not in the document]
Comment (vutiendanhvgu01) [reply src/containers/Detail/Detail.tsx]: I have fixed

File: src/containers/Home/styles.scss [hunk 5]
```diff
@@ -287,6 +289,62 @@
...
+      img{
+        width: 100%;
+        border-radius: 5px;
+
+      }
+    }
+    .job-detail__description{
+      width: 40%;
+      padding: 40px 50px;
+      h3{
```
[the remark this answers is not in the document]
Comment (vutiendanhvgu01) [reply src/containers/Home/styles.scss]: I have fixed

File: src/containers/JobComponent/JobComponent.tsx [hunk 6]
```diff
@@ -12,35 +13,37 @@ const JobComponent: FC<Props> = (props) => {
...
-        <div className="job" data-testid={`job${index}`} key={job.id}>
-          <div className="job__content" data-testid="job__content">
-            <div className="content__image" data-testid="content__image">
-              <img
-                className="content__image-random"
-                data-testid="content__image-random"
-                src={job.image}
-                alt="https://loremflickr.com/640/480/food"
-              />
+        <NavLink to={`/detail/${job.id}`} style={{ color: "black" }}>
```
[the remark this answers is not in the document]
Comment (vutiendanhvgu01) [reply src/containers/JobComponent/JobComponent.tsx]: I have fixed

File: src/redux/reducer/JobReducer.tsx [hunk 7]
```diff
@@ -40,5 +46,14 @@ export const jobAPI = () => {
...
+    console.log(result.data);
```
[the remark this answers is not in the document]
Comment (vutiendanhvgu01) [reply src/redux/reducer/JobReducer.tsx]: I have fixed

File: src/redux/reducer/JobReducer.tsx (hunk 1 above)
Comment (vutiendanhvgu01) [reply src/redux/reducer/JobReducer.tsx]: I have fixed

**Pull request merged**